ca/ca_test.go

```diff
@@ -70,19 +78,30 @@ func TestCertCreation(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// Test writing and reading our certificate to and from disk
-	certPath := path.Join(os.TempDir(), "cert.pem")
-	err = util.WriteCertFile(certPath, cc.Raw)
+	// Test reading our certificate from disk
+	testCa := Ca{
+		cfgPath: tempDir,
+	}
+	err = testCa.Load(string(jwa.ES256))
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	readCert, err := util.ReadCertFile(certPath)
+	if string(ca.CaCertBytes) != string(testCa.CaCertBytes) {
+		t.Fatal("failed reading CA cert bytes from disk")
+	}
+
+	ecPub, err := x509.MarshalPKIXPublicKey(ca.pksk.Public())
 	if err != nil {
-		t.Fatal(err)
+		t.Fatal("error marshalling public key")
```

Comment:
Might it be beneficial to include the returned `err` in the fatal log message?

Reply:
Since this is a test `err` is actually much more useful than the string since the output gives you the line number. Fixed in https://github.com/openpubkey/openpubkey/pull/79/commits/64bfdaf938062a21df12a9b3ac2068d075b7dd6d